### DISCUSSION TITLE
Skydimo

### DISCUSSION THREAD

**Discussion by m4jki** [awawa-dev/HyperHDR#999] [Q&A] [answered]
hi, can i use this soft with Skydimo ambient light strip?

https://www.skydimo.com/index_en.html

**Comment by awawa-dev** [comment]
Yes, you can with HyperHDR 21 (in development, installers available in the Github Action tab): https://github.com/awawa-dev/HyperHDR/issues/934#issuecomment-2453275828

**Comment by m4jki** [comment]
ok, i have a LED strip from skydimo, newest beta of HyperHDR but i don't know how to configure it. i created instance, add hardware, the lights turn on but there is no effect. remote control not working. dont know what to do next...

on windows it's working great with skydimo software.

no errors in logs also...
[skydimo.txt](https://github.com/user-attachments/files/18039842/skydimo.txt)

**Comment by m4jki** [comment]
![Zrzut ekranu z 2024-12-06 16-11-05](https://github.com/user-attachments/assets/ceafda8e-e969-45e4-accd-a4defecbe00c)
![Zrzut ekranu z 2024-12-06 16-10-58](https://github.com/user-attachments/assets/900f72cb-7d5d-4604-9cb6-c5481fd6040d)
![Zrzut ekranu z 2024-12-06 16-10-52](https://github.com/user-attachments/assets/ed06087e-c652-43c0-8a22-5782cbd99761)

**Comment by awawa-dev** [comment]
I don't know why it doesn't work either. I don't use those LEDs.
Did you read the comment from PR: the user used much lower baud speed.
Why you are running HyperHDR as root? What's the system?

**Reply by m4jki** [reply]
I'll try with lower speed. 

System is manjaro. And root is needed for writing configuration to device. Without root it not start at all

**Reply by awawa-dev** [reply, marked as answer]
Serial port does not need special rights unless the system is somehow misconfigured and the ordinary user has no right to /dev/tty* but I havent met such situation before. The Skydimo controler has UART that is 2Mb speed capable but the manufacturer probably trim the speed in the Arduino program to 115200. Anyway Arduino is not able to maintain such high speed 2Mb and LEDs at the same time...

**Reply by m4jki** [reply]
> The Skydimo controler has UART that is 2Mb speed capable but the manufacturer probably trim the speed in the Arduino program to 115200.

After changing to 115200 baud, it works perfectly. I also added the user to the appropriate groups, and now it can be run without root privileges.

thx!

**Comment by cahotts** [comment]
Hey @m4jki I have an skydimo too, but the only thing I can make it do, it's to show the default colors effects, but didn't working for screen capture, I will review anyway my configurations again, but any advice you can share to properly configure this? 
Os. Fedora 42
HyperHDR 22.0.0.0beta0

**Reply by awawa-dev** [reply]
If the effects are working then this is not related to Skydimo but your software grabber configuration (probably Wayland grabber) that you must enable first in HyperHDR and give it permission to capture the screen in the system dialog that should pop-up.

**Reply by cahotts** [reply]
thanks for the advice! It's working now :)

**Reply by GridyushkoF** [reply]
@cahotts i also use fedora and I encountered this problem: effects works, screen capture no. Can you tell me how you solved this problem

**Reply by cahotts** [reply]
@GridyushkoF - I just delete everithing related to HDR and compile it again from scratch, then I notice I have provide the capture video the right permission, but somehow the HDR software didn't recongnize it, so that's why I compile the software to fit my current config
